docs/index.md

```diff
@@ -1326,7 +1326,13 @@ $ mocha --reporter list --growl
 
 ## The `test/` Directory
 
-By default, `mocha` looks for the glob `./test/*.js`, so you may want to put your tests in `test/` folder. If you want to include sub directories, pass the `--recursive` option.
+By default, `mocha` looks for the glob `./test/*.js`, so you may want to put your tests in `test/` folder. Use the field `directories.test` in package.json to overwrite the default folder. If you want to include sub directories, pass the `--recursive` option.
+
```

Comment:
Confusing the way you added into the middle of existing paragraph here. Just make a new paragraph instead. Put `package.json` in double quotes and append the word 'file'. "Overwrite" is incorrect (and _scary_) -- maybe just say "use an alternate directory".

Reply:
ok, done